app/controllers/staff/reports_submissions_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Staff::ReportsSubmissionsController < Staff::BaseController
```

Comment:
I'd usually write this with `new` and `create` actions because you're "creating a new report submission", even if that doesn't exactly map to the way the underlying model works.

Of course, there's a question around how we implement additional changes in state - having a separate controller for each possible change could get complicated 🤔 

Happy to chat about this!

Reply:
This is definately something I would appreciate talking through as we have two other state changes to add next!

Reply:
I think it makes sense to have a distinct controller for changing the report's state, as it's a distinct type of change from updating all the other aspects of a report. However we can probably do with a single controller for all state changes, rather than one controller per state.